meta-ibm/meta-witherspoon/recipes-phosphor/power/witherspoon-pfault-analysis_git.bb

```diff
@@ -30,10 +31,8 @@ SEQ_MONITOR_FMT = "../${SEQ_MONITOR_SVC}:${CHASSIS_ON_TGT}.wants/${SEQ_MONITOR_S
 SEQ_PGOOD_SVC = "pseq-monitor-pgood.service"
 SEQ_PGOOD_FMT = "../${SEQ_PGOOD_SVC}:${CHASSIS_ON_TGT}.wants/${SEQ_PGOOD_SVC}"
 
-SYSTEMD_SERVICE_${PN}_append_ibm-ac-server += "${SEQ_MONITOR_SVC} ${SEQ_PGOOD_SVC}"
-SYSTEMD_LINK_${PN}_append_ibm-ac-server += "${SEQ_MONITOR_FMT} ${SEQ_PGOOD_FMT}"
-SYSTEMD_SERVICE_${PN}_append_mihawk += "${SEQ_MONITOR_SVC} ${SEQ_PGOOD_SVC}"
-SYSTEMD_LINK_${PN}_append_mihawk += "${SEQ_MONITOR_FMT} ${SEQ_PGOOD_FMT}"
```

Comment:
Why remove this? Should you add `SYSTEMD_LINK_${PN}_append_mowgli` and `SYSTEMD_SERVICE_${PN}_append_mowgli` instead?

Reply:
My original thought was that "ibm-ac-server", "mihawk" and "mowgli" all need to use these two services, so I made this update to make the bb file to be a little cleaner.
Sorry about that, I have changed it back.